gradle.properties

```diff
@@ -3,8 +3,7 @@ springVersion=4.3.3.RELEASE
 springBootVersion=1.4.1.RELEASE
 jerseyVersion=2.24
 grizzlyVersion=2.3.28
-cyclopsReactVersion=2.0.0-FINAL
```

Comment:
Looks like this is needed in some of .build files ( micro-async-data-loader/build.gradle, micro-async-data-writer/build.gradle,  micro-core/build.gradle, micro-jmx-metrics/build.gradle, micro-s3/build.gradle)

Reply:
fixed